### PR TITLE
Avatar and alert

### PR DESCRIPTION
**Pull request by MuskanChhatrasal**

### CODE REVIEW

**Review by aayushjain-code — COMMENTED** [review]
LGTM keep going!

**Comment by harshkotkar5557** [comment]
Looks goods. go for merge

**Review by harshkotkar5557 — COMMENTED** [review]
All looks good to me.

File: Layouts/navbar.css [hunk 1]
```diff
@@ -2,24 +2,49 @@
 
 .logo {
   color: var(--primary-color);
-  margin-left: 60px;
+  margin-left: 50px;
```
Comment (harshkotkar5557) [inline Layouts/navbar.css:5]: you use rem or em

**Review by harshkotkar5557 — COMMENTED** [review]
Very think looks to me. You add preview and go for merge

File: Layouts/navbar.css (hunk 1 above)
Comment (MuskanChhatrasal) [reply Layouts/navbar.css:5]: Done!! Thankyou

**Pull request merged**